simple-weather/simple-weather/Model/CurrentWeather.swift

```diff
@@ -9,40 +9,39 @@
 import Foundation
 
 class CurrentWeather {
+    var cityName: String!
```

Comment:
# **`!`**, we do not force unwrap
Either add default value `var cityName = ""` or have values as optionals `var cityName: String?`, optional for our case, to satisfy having uninitialised values in class.